atomicapp/constants.py

```diff
@@ -61,3 +61,5 @@
 PROVIDER_CONFIG_KEY = "providerconfig"
 
 PROVIDERS = ['docker', 'kubernetes', 'openshift']
+PROVIDER_API_KEY = "providerapi"
```

Comment:
I immediately imagined a token while reading `PROVIDER_API_KEY` so the name is probably not good